src/cms/urls.py

```diff
@@ -303,6 +312,27 @@
             ]
         ),
     ),
+    url(
+        r"^activation/",
+        include(
+            [
+                url(
+                    r"^(?P<uidb64>[0-9A-Za-z]+)/(?P<token>.+)/$",
+                    auth_views.PasswordResetConfirmView.as_view(
+                        template_name="authentication/account_activation_form.html",
+                        success_url=reverse_lazy("activation_complete"),
+                        form_class=ActivationForm,
```

Comment:
Cool idea to use the inbuilt view for this!

Reply:
Inspired by the password reset process:smiley: